docs/references/general/grid.md

```diff
@@ -0,0 +1,104 @@
+| *stid*      |  **None**       |  *[numstid]*     | ***short*** | A list of of numeric station IDs that provided data for the record |
+| *channel*   |  **None**       |  *[numstid]*     | ***short*** | A list of channel numbers associated to the station id the record |
+| *nvec*      | **None** | *[numstid]*  | ***short*** | Number of velocity vectors for each station|
+| *freq*      | *kHz* | *[numstid]* | ***float*** | Transmitted frequency for each radar |
+| *major.revision* | **None** | *[numstid]* | ***short*** | Major `make_grid` version number                    |
+| *minor.revision* | **None** | *[numstid]* | ***short*** | Minor `make_grid` version number per station number |
+| *program.id*     | **None** | *[numstid]* | ***short*** | Control program ID per station number 
+| *noise.mean*      | **None** | *[numstid]* | ***float*** | Mean noise for each radar |
+| *noise.sd*        | **None** | *[numstid]* | ***float*** | Noise Standard deviation per station number |
+| *gsct*            | **None** | *[numstid]* | ***short*** | A list of groundscatter flags associated to each radars data for the given record |
+| *v.min*           | *m/s* | *[numstid]* | ***float*** | Minimum velocity threshold |
+| *v.max*           | *m/s* | *[numstid]* | ***float*** | Velocity maximum threshold per station number |
+| *p.min*           | **None** | *[numstid]* | ***float*** | Power minimum threshold per station number |
+| *p.max*           | **None** | *[numstid]* | ***float*** | Power maximum threshold per station number |
+| *w.min*           | *m/s*  | *[numstid]* | ***float*** | Spectral width minimum threshold per station number |
+| *w.max*           | *m/s*  | *[numstid]* | ***float*** | Spectral width maximum threshold per station number |
+| *ve.min*          | *m/s*  | *[numstid]* | ***float*** | Velocity error minimum threshold per station number |
+| *ve.max*          | *m/s*  | *[numstid]* | ***float*** | Velocity error maximum threshold per station number |
+| *vector.mlat*     | *degrees* | *[numv]* | ***float*** | Magnetic Latitude per vector point |
+| *vector.mlon*     | *degrees* | *[numv]* | ***float*** | Magnetic Longitude  per vector point |
+| *vector.kvec*     | *degrees*  | *[numv]*   | ***flost*** | Magnetic Azimuth per vector point |
+| *vector.stid*     | **None**  | *[numv]*   | ***short*** | Station identifier per vector point|
+| *vector.channel*  | **None**  | *[numv]*   | ***short*** | Channel number per vector point |  
+| *vector.index*    | **None**  | *[numv]*   | ***int***   | Grid cell index |
+| *vector.vel.median* | *m/s* | *[numv]*   | ***float*** | Velocity medians per vector point|
+| *vector.vel.sd*     | *m/s*   | *[numv]*   | ***float*** | Velocity standard deviation per vector point|
+| *vector.pwr.median* | *dB*  | *[numv]*   | ***float*** | Power median per vector point|
```

Comment:
```suggestion
| *vector.pwr.median* | *dB*  | *[numv]*   | ***float*** | Weighted mean power per vector point|
```
Updated based on @egthomas comment on what median means.

Reply:
Strongly recommend removing "per vector point". The descriptions used [here](https://github.com/SuperDARN/rst/blob/master/codebase/superdarn/src.lib/tk/gtable.2.0/src/gtable.c#L169-L182) is much clearer

Reply:
Updated now.  Sorry I'm coming in very late on this and maybe causing confusion!

Reply:
It doesn't look like the removal of "per vector point" (or "per station number") ever took place.

Reply:
@egthomas Yep, just noticed that too. I won't merge this PR just yet -- clearly needs one final check!